sdk/src/Services/DynamoDBv2/Custom/DocumentModel/JsonUtils.cs

```diff
@@ -352,7 +352,7 @@ internal static void WriteJson(DynamoDBEntry entry, JsonWriter writer, DynamoDBE
             var ddbNull = entry as DynamoDBNull;
             if (ddbNull != null)
             {
-                writer.Write(null);
+                writer.Write((string)null);
```

Comment:
this was just to fix a build problem for the nullable value types in collections. Bo and I talked about making this change.

Reply:
Yes, it was one I missed from this commit which we PRed: https://github.com/aws/aws-sdk-net/pull/3243/commits/b19a0105bfa967cbd04f57d38709c6c699db1aff